Content.Server/Lock/LockComponent.cs

```diff
@@ -0,0 +1,63 @@
+using Content.Server.Lock;
+using Content.Shared.ActionBlocker;
+using Content.Shared.Interaction;
+using Content.Shared.Interaction.Helpers;
+using Content.Shared.Verbs;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Localization;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.Storage.Components
+{
+    /// <summary>
+    /// Allows locking/unlocking, with access determined by AccessReader
+    /// </summary>
+    [RegisterComponent]
+    public class LockComponent : Component
+    {
+        public override string Name => "Lock";
+
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("locked")] public bool Locked { get; set; } = true;
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("unlockingSound")] public string UnlockSound { get; set; } = "/Audio/Machines/door_lock_off.ogg";
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("lockingSound")] public string LockSound { get; set; } = "/Audio/Machines/door_lock_off.ogg";
```

Comment:
replace these with SoundSpecifier, and IMO they should be nullable since you might not want to play any sound at all

Reply:
done